lib/pptx_masher/slide.rb

```diff
@@ -50,7 +50,12 @@ def replace_media(dest, src_io)
       extension = dest.split('.').last
       new_name = "image#{presentation.media_count+1}.#{extension}"
       gsub_file rels_path, dest, new_name
-      File.open("#{presentation_path}/ppt/media/#{new_name}", 'wb') {|f| f << src_io.read }
+      File.open("#{presentation_path}/ppt/media/#{new_name}", 'wb') do |f| 
+        buffer = ''
+        while src_io.read(1048576, buffer)
+          f.puts buffer
```

Comment:
@layerssss i just caught a major bug! This also explains why we were getting weird issues in product. `puts` adds a `\n`!!!

Fortunately I discovered `IO.copy_stream` which does the same thing.